src/main/java/org/killbill/billing/plugin/adyen/api/AdyenPaymentPluginApi.java

```diff
@@ -185,39 +188,41 @@ protected String getPaymentMethodId(final AdyenPaymentMethodsRecord paymentMetho
     @Override
     public PaymentTransactionInfoPlugin authorizePayment(final UUID kbAccountId, final UUID kbPaymentId, final UUID kbTransactionId, final UUID kbPaymentMethodId, final BigDecimal amount, final Currency currency, final Iterable<PluginProperty> properties, final CallContext context) throws PaymentPluginApiException {
         return executeInitialTransaction(TransactionType.AUTHORIZE,
-                                         new TransactionExecutor<PurchaseResult, RuntimeException>() {
-                                             @Override
-                                             public PurchaseResult execute(final Long amount, final PaymentData paymentData, final OrderData orderData, final UserData userData, final String termUrl, final SplitSettlementData splitSettlementData) {
-                                                 return adyenConfigurationHandler.getConfigurable(context.getTenantId()).authorise(amount, paymentData, orderData, userData, termUrl, splitSettlementData);
-                                             }
-                                         },
-                                         kbAccountId,
-                                         kbPaymentId,
-                                         kbTransactionId,
-                                         kbPaymentMethodId,
-                                         amount,
-                                         currency,
-                                         properties,
-                                         context);
+                new TransactionExecutor<PurchaseResult>() {
```

Comment:
Similarly, it doesn't look like there is a functional change here (only indentation & materialization of `AdyenPaymentServiceProviderPort`)? Can we tackle them in another PR?